### DISCUSSION TITLE
Price history data?

### DISCUSSION THREAD

**Discussion by Meruman** [hugoattal/OrbitalMarket#37] [Q&A] [answered]
Hi, I was wondering how do you get the price history data of the assets? Where is that data coming from? Is there any API from epic games that sends this?

Thanks! :)

**Comment by hugoattal** [comment, marked as answer]
Hey! Nop, the price is updated every 24h, and Orbital Market keeps track of it.

Keep in mind that this is the base price, so no discount is taken into account.

**Reply by Meruman** [reply]
okok thanks!
